test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -68,8 +68,8 @@ const EventEmitter = require('events');
 }
 
 {
-  const listen1 = function listen1() {};
-  const listen2 = function listen2() {};
+  const listen1 = () => {};
```

Comment:
@aqrln In that case, I imagine explicitly defining `const noop = () => {};` in the test itself would make it clear that everything is intentionally using the same function object and that it's not a mistake.